### DISCUSSION TITLE
Time Measurement

### DISCUSSION THREAD

**Discussion by mnmoron** [OpenHantek/OpenHantek6022#216] [Q&A] [answered]
Is there a way to get the measurement cursors to have better "resolution"? When I drag the cursors, they snap to horizontal locations that are somewhat larger than the individual samples.

**Comment by Ho-Ro** [comment, marked as answer]
If you grab the cursor inside of the scope screen you can drag it with small steps, if you grab it by the number it will snap to 1% of the screen width.

**Reply by mnmoron** [reply]
Excellent. That works. Thank you.
